docs/source/starter/installation.rst

```diff
@@ -0,0 +1,55 @@
+.. _installation:
+
+############
+Installation
```

Comment:
This isn't about installation. Maybe let's rename this Lightning Coverage

Reply:
it's about installation. Lightning coverage is just a small part to indicate whether users' env will be supported with PL or not.